doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.postprocess.TransformSourceTableTask.rst

```diff
@@ -8,8 +8,8 @@ TransformSourceTableTask
 ``TransformSourceTableTask`` transforms the full-width source table
 (a ``source`` dataset) to a narrower Source Table (a ``sourceTable`` dataset)
 as specified by the Data Products Definition Document (`DPDD <https://lse-163.lsst.io>`).
-It extracts, transforms, and renames columns per a yaml specification, by default the
-`Source.yaml` in obs_package/policy.   Inputs and outpus are both per-detector.
+It extracts, transforms, and renames columns per a yaml specification, by default the `schemas/Source.yaml` in this package.
```

Comment:
Not too sure of the `.rst` style guide about line-lengths - did you move this here rather than the line below to try and make it one sentence per line? If so, that makes sense.

Reply:
The dev guide recommends soft-wrapping in .rst files, with one sentence per line:

https://developer.lsst.io/restructuredtext/style.html#text-wrapping